time/time_test.go

```diff
@@ -0,0 +1,24 @@
+package time
+
+import (
+	"testing"
+)
+
+func TestUnMarshalText(t *testing.T) {
+	dummyTime := Time{}
+
+	goodTimeFormats := []string{
+		"2006-01-02T15:04:05MST",
+		"2006-01-02T15:04:05",
+	}
+	for _, goodTime := range goodTimeFormats {
+		if dummyTime.UnmarshalText([]byte(goodTime)) != nil {
```

Comment:
for some extra sanity, can you split the formats and actual values apart, so the tests ensures that multiple times of the given format work?